codegen/type_converter.go

```diff
@@ -779,14 +870,14 @@ type FieldAssignment struct {
 
 // IsTransform returns true if field assignment is not proxy
 func (f *FieldAssignment) IsTransform() bool {
-	var (
```

Comment:
I think it's better we do `trimName` that trims all the wired prefix and do 
`trimed(inName) != trimed(outName)`

this logic here is hard to read now

Reply:
Okay, I'll try to make it more obvious it's doing that, but there's still the prefix check too 😛